modules/elastic-search/main.tf

```diff
@@ -0,0 +1,62 @@
+module "elastic_search" {
+  source  = "lgallard/elasticsearch/aws"
+  version = "0.14.1"
+
+  domain_name           = var.domain_name
+  elasticsearch_version = var.es_version
+
+  cluster_config = {
+    dedicated_master_enabled = var.dedicated_master_enabled
+    instance_count           = var.instance_count
+    instance_type            = var.instance_type
+    zone_awareness_enabled   = var.zone_awareness_enabled
+    availability_zone_count  = var.availability_zone_count
+  }
```

Comment:
Wouldn't it better to have a singe variable with object type named like "cluster_config" instead of these separate variables?